### DISCUSSION TITLE
slack channel link doesn't work

### DISCUSSION THREAD

**Discussion by mcncyo** [siderolabs/talos#8705] [General]
your slack channel link doesn't work anymore.

**Comment by steverfrancis** [comment]
Thanks for letting us know!
Until we fix the link in the docs, you can use https://join.slack.com/t/taloscommunity/shared_invite/zt-2i72vkmxv-PqG~PemjdjmApSivYRUCPg

**Reply by andyetifish** [reply]
The link is no longer active

**Reply by steverfrancis** [reply]
Which link? The one in the docs? Or just in this issue?

**Comment by andyetifish** [comment]
The one here and the docs

**Reply by steverfrancis** [reply]
The docs link is https://slack.dev.talos-systems.io/
which seems to work for me in incognito window.
It links to 
https://join.slack.com/t/taloscommunity/shared_invite/zt-2ol2h4vz6-MmdYMdMwq3yeSq~w2fs42g
WHat are you seeing?

**Reply by Batyray** [reply]
Hello @steverfrancis The link seems like it is still not working 
![image](https://github.com/user-attachments/assets/c43bb303-2a5b-4905-9c13-d5671bd18040)

**Reply by steverfrancis** [reply]
Updated - should work now, thanks

**Comment by kylepl** [comment]
Note that this seems to be broken again.

**Reply by steverfrancis** [reply]
Thanks for the heads up. We will fix, but in the meantime you can use:
https://inviter.co/sidero-labs-community

**Comment by steverfrancis** [comment]
What was the error?
DId you get in?
